.github/workflows/galaxy-publish-check.yml

```diff
@@ -0,0 +1,41 @@
+name: Galaxy Publish Check
+on:
+  # Run publish check against all pushes (direct commits, also merged PRs), Pull Requests
+  push:
+    branches: [ main, release-* ]
+  pull_request:
+    branches: [ main, release-* ]
+  # manual trigger
+  workflow_dispatch:
+
+env:
+  NAMESPACE: ocmplus
+  COLLECTION_NAME: cm
+  COLLECTION_VERSION: 0.0.1 
```

Comment:
do we want to use hard coded version for all branches?

Reply:
I think it's good until we start to increase the version 👍

Reply:
Or at least until we start versioning branches